src/main/java/it/pagopa/ecommerce/commons/domain/pojos/BaseTransactionWithCompletedAuthorization.java

```diff
@@ -15,8 +15,8 @@
  * <p>
  * This POJO is needed to abstract inheritance and operations over both
  * {@link BaseTransactionAuthorized} and
- * {@link BaseTransactionWithFailedAuthorization}, and should not be extended by
- * an aggregate class.
+ * {@link BaseTransactionWithRequestedAuthorization}, and should not be extended
+ * by an aggregate class.
```

Comment:
Now that we have collapsed `Authorized` and `AuthorizationFailed` into a single state this comment can be removed (along with `BaseTransactionAuthorized` which is just a clone of this semantically)

Reply:
Fixed in https://github.com/pagopa/pagopa-ecommerce-commons/pull/31/commits/8656938d3b59201b9d59d42b17c77f81d570c50c